src/Workleap.Extensions.Mongo.Ephemeral.v2/Workleap.Extensions.Mongo.Ephemeral.v2.csproj

```
@@ -0,0 +1,35 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>net8.0;netstandard2.0</TargetFrameworks>
+    <IsPackable>True</IsPackable>
+    <AssemblyName>Workleap.Extensions.Mongo.Ephemeral</AssemblyName>
+    <PackageId>Workleap.Extensions.Mongo.Ephemeral.v2</PackageId>
+    <RootNamespace>Workleap.Extensions.Mongo.Ephemeral</RootNamespace>
+    <PackageReadmeFile>README.md</PackageReadmeFile>
+    <DefineConstants>$(DefineConstants);MONGODB_V2</DefineConstants>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="..\Workleap.Extensions.Mongo.Ephemeral\**\*.cs" />
+    <Compile Remove="..\Workleap.Extensions.Mongo.Ephemeral\bin\**\*.cs" />
+    <Compile Remove="..\Workleap.Extensions.Mongo.Ephemeral\obj\**\*.cs" />
```

I think you can use `Exclude` to prevent adding extra items.
Not sure globbing `{bin,obj}` works, but you can create 2 globs if needed (`;`-separated)

```suggestion
    <Compile Include="..\Workleap.Extensions.Mongo.Ephemeral\**\*.cs" Exclude="..\Workleap.Extensions.Mongo.Ephemeral\{bin,obj}\**\*.cs" />
```

Another strategy would be to use [Artifacts output layout[(https://learn.microsoft.com/en-us/dotnet/core/sdk/artifacts-output?WT.mc_id=DT-MVP-5003978), so you don't need to exclude any item.

---

I went with artifacts, so no more need to remove obj/ C# files (there wasn't any in bin/)